### ISSUE TITLE
nil value error

### ISSUE THREAD

**Issue by bdarcus** [pandoc-scholar/pandoc-scholar#2]
I have the following Makefile:
```
ARTICLE_FILE        = book.md

PANDOC_LATEX_OPTIONS  = --latex-engine=lualatex
PANDOC_LATEX_OPTIONS += --csl=peerj.csl
PANDOC_LATEX_OPTIONS += --filter=pandoc-citeproc
PANDOC_LATEX_OPTIONS += -M fontsize=10pt
PANDOC_LATEX_OPTIONS += -M classoption=memoir

PANDOC_HTML_OPTIONS   = --toc
PANDOC_EPUB_OPTIONS   = --toc

DOCX_REFERENCE_FILE   = book.docx
ODT_REFERENCE_FILE    = book.odt

PANDOC_SCHOLAR_PATH = /home/bruce/Code/thirdparty/pandoc-scholar
include $(PANDOC_SCHOLAR_PATH)/Makefile
```
... and the following book.md:

```
% Book Title
% John Doe
% 2017

# Introduction

Something

## Subheading

More.

# Another

Something else.
```
... but get this error:
```
 make
pandoc --smart \
       -t /home/bruce/Code/thirdparty/pandoc-scholar/writers/affiliations.lua \
       --output outfile.enriched.json book.md
pandoc: user error (/home/bruce/Code/thirdparty/pandoc-scholar/scholarly-metadata/panlunatic.lua:126: attempt to concatenate a nil value)
make: *** [/home/bruce/Code/thirdparty/pandoc-scholar/Makefile:26: outfile.enriched.json] Error 1
```

I thought maybe the process was expecting a YAML metadata block, but that appears not to be the issue.

**Comment by tarleb** [comment]
Thanks for the report! It appears that an old version of an internally used library is loaded, probably due to an issue in the way the distribution archive is packed.  Could you try replacing the content of file `scholarly-metadata/panlunatic.lua` with the [current version of that library](https://raw.githubusercontent.com/tarleb/panlunatic/master/src/panlunatic.lua)?  I'm going to update the respective files asap.

**Comment by tarleb** [comment]
Should be fixed in the newly released version 1.2.2

**Comment by bdarcus** [comment]
Yes, that's fixed; thanks.

I did run into a further error with those `*REFERENCE_FILE` variables, but that may be just a minor documentation issue.
